application/src/main/java/com/softserve/lv460/application/entity/TelegramActivation.java

```diff
@@ -0,0 +1,26 @@
+package com.softserve.lv460.application.entity;
+
+import lombok.*;
+
+import javax.persistence.*;
+import java.security.SecureRandom;
+import java.time.LocalDateTime;
+
+@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class TelegramActivation {
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private Long id;
+
+  private String token = String.format("%05d", new SecureRandom().nextInt(1000000));
```

Comment:
don't like this values calculated inline. Maybe make a private constructor with some static init method?